.github/workflows/client-celo.yml

```diff
@@ -67,30 +75,30 @@ jobs:
             gotestsum -- -tags=celo,musl ./...
 
       - name: Login to Google Container Registry
-        if:  |
+        if: |
           github.ref == 'refs/heads/master'
             && github.event_name != 'pull_request'
         uses: docker/login-action@v1
         with:
-          registry: ${{ secrets.GCR_REGISTRY_URL }}
+          registry: ${{ env.GCR_REGISTRY_URL }}
           username: _json_key
           password: ${{ secrets.KEEP_TEST_GCR_JSON_KEY }}
 
       - name: Build and publish Docker Runtime Image
         uses: docker/build-push-action@v2
         env:
           IMAGE_NAME: 'keep-ecdsa-celo'
-          GOOGLE_PROJECT_ID: ${{ secrets.KEEP_TEST_GOOGLE_PROJECT_ID }}
         with:
           # GCR image should be named according to following convention:
           # HOSTNAME/PROJECT-ID/IMAGE:TAG
           # We don't use TAG yet, will be added at later stages of work on RFC-18.
-          tags: ${{ secrets.GCR_REGISTRY_URL }}/${{ env.GOOGLE_PROJECT_ID }}/${{ env.IMAGE_NAME }}
+          tags: ${{ env.GCR_REGISTRY_URL }}/${{ env.GOOGLE_PROJECT_ID }}/${{ env.IMAGE_NAME }}
           labels: revision=${{ github.sha }}
           build-args: |
             HOST_CHAIN=celo
             REVISION=${{ github.sha }}
           # VERSION= ? TODO: Configure version, sample: 1.7.6
           push: |
-            ${{ github.ref == 'refs/heads/master'
-              && github.event_name != 'pull_request' }}
\ No newline at end of file
+            ${{ (github.ref == 'refs/heads/master'
+              && github.event_name != 'pull_request')
+              || github.ref == 'refs/heads/rfc-18/load-env-var' }}
```

Comment:
Can we remove it already?

Reply:
Done, see 7c45d4f3dea23cfcdfb84431e543788a41caf425.